src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('table');
+const tbody = document.querySelector('tbody');
+const tr = tbody.querySelectorAll('tr');
+const row = [...tr];
+
+table.addEventListener('click', (e) => {
+  const th = e.target.closest('th');
+
+  if (!th || !table.contains(th)) {
+    return;
+  }
+
+  sortTable(th.cellIndex, th.innerHTML);
```

Comment:
It looks easier for me)
```suggestion
  if (th && table.contains(th)) {
     sortTable(th.cellIndex, th.innerHTML);
  } 
```